Refactors text input error styles for #219

Martskin opened the pull request:
- Updates text input error styles.
- Adds notification color variables.
- Adds success text input styles.
- Add help text element for text forms.

cehfisher commented:
thanks for this update @Martskin! I finally got some time to review your PRs...a couple of quick tips for your next PR:
- make changes (as you did)
- run `npm run gulp` (this adds your source changes to the dist folder)
- if you want, you can update the demo by running the commands `npm run gulp clean:docs`  (Removes old code from /docs except for index and KSS settings)
`npm run gulp move:docs` (Move items from /dist to /docs)

but these are things I can fix on the PR review too, if easier for you :)
thanks again,
Carie

Martskin commented:
Thanks @cehfisher ! I saw the `clean:docs` and `move:docs` scripts in the `CONTRIBUTING.md` file but when I ran in in my branches there were several files modified that I hadn't touched so I decided to not commit the moved docs:

```console
modified:   docs/css/videos.css
modified:   docs/style-guide/index.htm
Untracked files:
docs/assets/px-video-sprite.svg
```
